test/prometheus/prometheus-operator-serviceMonitor.yaml

```diff
@@ -10,12 +10,13 @@ metadata:
   namespace: monitoring
 spec:
   endpoints:
-    - bearerTokenFile: /var/run/secrets/kubernetes.io/serviceaccount/token
-      honorLabels: true
-      port: https
-      scheme: https
-      tlsConfig:
-        insecureSkipVerify: true
+    - basicAuth:
```

Comment:
added this to show that auth works since metrics tests pass, shall I revert it ?

Reply:
I am ok with keeping it like this